### DISCUSSION TITLE
忍者神龟:施莱德的复仇

### DISCUSSION THREAD

**Discussion by Weedinfire** [Gamuxorg/bbs#297] [投递和请求游戏]
游戏名称：忍者神龟:施莱德的复仇
游戏名称(英文)：Teenage Mutant Ninja Turtles: Shredder's Revenge
下载地址 :（如果有的话）：

**Comment by zgldh** [comment]
https://www.linuxgame.cn/%e5%bf%8d%e8%80%85%e9%be%9f%e6%96%bd%e8%8e%b1%e5%be%b7%e7%9a%84%e5%a4%8d%e4%bb%87
